scripts/widgets/mz-search-pagination-dd.js

```diff
@@ -0,0 +1,168 @@
+﻿
+define(["jquery", "underscore"], function ($, _) {
+
+    $.fn.mzdropdown = function (options) {
```

Comment:
It's very confusing how many different names you have used for your single component. Is it `mzdropdown` or `mz-search-pagination-dd` or is it `mz-search-dd`?

I'd prefer to see all your file names, CSS class names, and such make sense. If you called everything `mozuDropdown` for example it would make more sense. Just like we call our grid `mozuGrid` instead of `mozuPaginatedSearchableGrid`.

Reply:
Done, changed name.